src/text-buffer.coffee

```diff
@@ -65,8 +64,6 @@ class TextBuffer
   @Patch: require('./patch')
   @newlineRegex: newlineRegex
 
-  Serializable.includeInto(this)
```

Comment:
Here I have removed `Serializable` so that we are allowed to pass arbitrary parameters to the `TextBuffer.prototype.serialize` method, such as the one to exclude marker layers from being serialized.